src/Tizen.NUI.Components/Controls/Menu.cs

```diff
@@ -640,5 +640,28 @@ private void CalculateScrimPosition()
                 Scrim.Position2D = new Position2D(-Position2D.X, -Position2D.Y);
             }
         }
+
+        /// <summary>
+        /// Initialize AT-SPI object.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override void OnInitialize()
+        {
+            base.OnInitialize();
+            SetAccessibilityConstructor(Role.PopupMenu);
+            AppendAccessibilityAttribute("sub-role", "Alert");
+            Show(); // calls AddPopup()
```

Comment:
Mr. kimcinoo pointed out whether this `Show()` is proper.
@jaehyun0cho 
Hello, Mr. jaehyun0cho
could you check this? I don't think this is required, because this is about ATSPI and ATSPI should not control the Component itself.

Reply:
@m-bialota How about moving Show() to the end of Post()?
Because Menu is supposed to be displayed when app calls Post().

Reply:
OK, I moved the call to Post(). it works well too :).